# Portuguese translation for enigma2 OBH-Core.
# Copyright (C) 2012 THE plugins'S COPYRIGHT HOLDER
# This file is distributed under the same license as the plugins package.
# Automatically generated, 2012.
#
msgid ""
msgstr ""
"Project-Id-Version: enigma2-obh-core\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-01-03 18:19+0000\n"
"PO-Revision-Date: 2018-10-31 22:04+0000\n"
"Last-Translator: \n"
"Language-Team: Black Hole Team\n"
"Language: pt_PT\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 2.2\n"
"X-Poedit-Basepath: .\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-SearchPath-0: .\n"

#: BackupManager.py:44 ImageManager.py:40
msgid "Daily"
msgstr "Diariamente"

#: BackupManager.py:44 ImageManager.py:40
msgid "Weekly"
msgstr "Semanalmente"

#: BackupManager.py:44 ImageManager.py:40
msgid "30 Days"
msgstr "30 Dias"

#: BackupManager.py:112 BackupManager.py:173 BackupManager.py:319
#: BackupManager.py:335 BackupManager.py:378 BackupManager.py:396
#: BackupManager.py:1027 ImageManager.py:349 ui.py:41
msgid "Backup manager"
msgstr "Gestor de Backup"

#: BackupManager.py:135
msgid "Restore"
msgstr "Repor"

#: BackupManager.py:136 ImageManager.py:105 SwapManager.py:189
msgid "Delete"
msgstr "Apagar"

#: BackupManager.py:148 BackupManager.py:150 BackupManager.py:294
#: BackupManager.py:296 ImageManager.py:118 ImageManager.py:120 ImageManager.py:286
#: ImageManager.py:288
msgid "Next backup: "
msgstr "Backup seguinte: "

#: BackupManager.py:148 BackupManager.py:294 ImageManager.py:118 ImageManager.py:286
msgid "%a %e %b  %-H:%M"
msgstr "%a %e %b  %-H:%M"

#: BackupManager.py:177 BackupManager.py:332 BackupManager.py:375
#: ImageManager.py:148 ImageManager.py:335
msgid "View progress"
msgstr "Ver progresso"

#: BackupManager.py:179 ImageManager.py:150
msgid "New backup"
msgstr "Novo backup"

#: BackupManager.py:223 ImageManager.py:222
msgid "The chosen location does not exist, using /media/hdd"
msgstr "O local escolhido não existe, a usar /media/hdd"

#: BackupManager.py:223
msgid "Select a backup to Restore:"
msgstr "Selecionar backup a Restaurar:"

#: BackupManager.py:232
msgid "Device: none available"
msgstr "Dispositivo: nenhum disponível"

#: BackupManager.py:232 BackupManager.py:246
msgid "Select a backup to restore:"
msgstr "Selecione um backup para restaurar:"

#: BackupManager.py:246 BackupManager.py:263 ImageManager.py:249 ImageManager.py:258
#: MountManager.py:189 MountManager.py:204 MountManager.py:224
msgid "Device: "
msgstr "Dispositivo: "

#: BackupManager.py:263 ImageManager.py:258
msgid "there is a problem with this device, please reformat it and try again."
msgstr ""
"há um problema com este dispositivo, por favor reformate-o e tente novamente."

#: BackupManager.py:303
msgid ""
"Are you sure you want to delete this backup:\n"
" "
msgstr ""
"Tem certeza que deseja excluir este Backup:\n"
" "

#: BackupManager.py:305 ImageManager.py:297
msgid "Remove confirmation"
msgstr "Confirmação para remover"

#: BackupManager.py:307
msgid "You have no backup to delete."
msgstr "Não tem backups para apagar."

#: BackupManager.py:349
msgid "You have no backups to restore."
msgstr "Não tem algum backup para restaurar."

#: BackupManager.py:351
msgid ""
"Backup in progress,\n"
"Please wait for it to finish, before trying again"
msgstr ""
"Backup em andamento, Por favor, espere que termine, antes de tentar novamente"

#: BackupManager.py:362 BackupManager.py:364 RestoreWizard.py:198
#: RestoreWizard.py:202
msgid "Sorry, but the file is not compatible with this image version."
msgstr "Desculpe, mas o arquivo não é compatível com esta versão da imagem."

#: BackupManager.py:367
msgid ""
"Are you sure you want to restore this backup:\n"
" "
msgstr ""
"Tem a certeza que prentende restaurar este Backup:\n"
" "

#: BackupManager.py:369
msgid "Restore Confirmation"
msgstr "Confirmação de restauro"

#: BackupManager.py:397 BackupManager.py:400 BackupManager.py:403
msgid "Restoring backup..."
msgstr "A restaurar o backup..."

#: BackupManager.py:406 BackupManager.py:409 BackupManager.py:1034
#: BackupManager.py:1037
msgid "Creating list of installed plugins..."
msgstr "A criar lista dos plugins instalados..."

#: BackupManager.py:412 BackupManager.py:415
msgid "Comparing against backup..."
msgstr "A comparar com o Backup..."

#: BackupManager.py:418 BackupManager.py:421
msgid "Restoring plugins..."
msgstr "A restaurar plugins..."

#: BackupManager.py:424 BackupManager.py:427
msgid "Restoring plugins, this can take a long time..."
msgstr "A restaurar plugins, isto pode demorar algum tempo..."

#: BackupManager.py:430
msgid "Rebooting..."
msgstr "A reiniciar..."

#: BackupManager.py:436
msgid "Do you want to restore your enigma2 settings ?"
msgstr "Deseja restaurar as configurações de enigma2?"

#: BackupManager.py:459
msgid "Sorry, but the restore failed."
msgstr "Desculpe, mas o restauro falhou."

#: BackupManager.py:478 RestoreWizard.py:173 RestoreWizard.py:282
msgid "A background update check is in progress, please try again."
msgstr ""
"Uma verificação de atualização em segundo plano está em andamento, por favor "
"tente novamente."

#: BackupManager.py:509
msgid "Sorry feeds are down for maintenance, Please try again later."
msgstr "Desculpe, os Feeds estão em manutenção, por favor tente mais tarde."

#: BackupManager.py:513
#, python-format
msgid ""
"Your %s %s is not connected to the internet, please check your network settings "
"and try again."
msgstr ""
"A sua %s %s não está ligada à Internet, verifique as configurações de rede e "
"tente novamente."

#: BackupManager.py:590
msgid "Do you want to restore your Enigma2 plugins ?"
msgstr "Deseja fazer um restauro dos Plugins Enigma2 ?"

#: BackupManager.py:602
msgid "Now skipping restore process"
msgstr "Saltar o processo de restauro"

#: BackupManager.py:639
msgid "Select files/folders to backup"
msgstr "Arquivos e pastas para Backup"

#: BackupManager.py:652 BackupManager.py:742 BackupManager.py:823
#: MountManager.py:299
msgid "Cancel"
msgstr "Cancelar"

#: BackupManager.py:653 BackupManager.py:743 MountManager.py:298
msgid "Save"
msgstr "Gravar"

#: BackupManager.py:684
msgid "Deselect"
msgstr "Desmarcar"

#: BackupManager.py:686
msgid "Select"
msgstr "Selecionar"

#: BackupManager.py:730
msgid "Select extra packages folder"
msgstr "Selecionar pasta de pacotes extra"

#: BackupManager.py:770
msgid "Select folder that contains plugins"
msgstr "Selecionar pasta que contém plugins"

#: BackupManager.py:796
msgid "Please enter a folder that contains some packages."
msgstr "Por favor introduza uma pasta que contém alguns pacotes."

#: BackupManager.py:824 RestoreWizard.py:119
msgid "OK"
msgstr "OK"

#: BackupManager.py:825
msgid "Choose files"
msgstr "Escolher arquivos"

#: BackupManager.py:826
msgid "Choose local ipk's folder"
msgstr "Escolher a pasta local de Ipk's"

#: BackupManager.py:847
msgid "Logs"
msgstr "Logs de depuração"

#: BackupManager.py:861
msgid "Backup created"
msgstr "Backup criado"

#: BackupManager.py:963
#, python-format
msgid ""
"Your %s %s is about to run a backup of your settings and detect your plugins,\n"
"Do you want to allow this?"
msgstr ""
"A sua %s %s está em vias de executar um backup das suas definições e detetar os "
"seus plugins,\n"
"Quer permitir isto?"

#: BackupManager.py:988 ImageManager.py:498
msgid "Enough retries, delaying till next schedule."
msgstr "Tentativas suficientes, adiar até ao próximo agendamento."

#: BackupManager.py:1028 BackupManager.py:1031
msgid "Starting..."
msgstr "A iniciar..."

#: BackupManager.py:1040 BackupManager.py:1043 BackupManager.py:1052
#: BackupManager.py:1055
msgid "Backing up files..."
msgstr "A executar Backup dos ficheiros..."

#: BackupManager.py:1046 BackupManager.py:1049
msgid "Preparing extra plugins..."
msgstr "A preparar plugins extra..."

#: BackupManager.py:1058
msgid "Backup Complete..."
msgstr "Backup completo..."

#: BackupManager.py:1108
msgid "The backup location does not have enough free space."
msgstr "O local de backup não tem espaço livre suficiente."

#: BackupManager.py:1148
msgid ""
"Plugin listing failed - e. g. wrong backup destination or no space left on backup "
"device"
msgstr ""
"Listagem de plugins falhou - i.e. destino do backup errado ou sem espaço no "
"dispositivo"

#: BackupManager.py:1209
msgid ""
"Backup failed - e. g. wrong backup destination or no space left on backup device"
msgstr ""
"Falha na cópia de segurança - i.e., destino errado ou sem espaço no dispositivo"

#: IPKInstaller.py:23 IPKInstaller.py:147
msgid "IPK installer"
msgstr "Instalador de IPK"

#: IPKInstaller.py:49 IPKInstaller.py:154 ImageManager.py:952 SwapManager.py:106
#: ui.py:65
msgid "Close"
msgstr "Fechar"

#: IPKInstaller.py:50 IPKInstaller.py:155
msgid "Install"
msgstr "Instalar"

#: IPKInstaller.py:75 IPKInstaller.py:94
msgid "Extra IPK's"
msgstr "IPK's Extra"

#: IPKInstaller.py:78
msgid ""
"It seems you have not setup an extra location, please set it up in the Backup "
"Manager setup menu "
msgstr ""
"Parece que não tem configurado um local extra, por favor, configure-o no menu do "
"Gestor de Backups "

#: IPKInstaller.py:80 IPKInstaller.py:84
msgid "Change location"
msgstr "Alterar localização"

#: IPKInstaller.py:82
msgid ""
"Sorry but that location does not exist or is not setup, please set it up in the "
"Backup Manager setup menu "
msgstr ""
"Desculpe, mas esse local não existe ou não está configurado, por favor configure-"
"o no menu de configuração do Gerenciador de backup "

#: IPKInstaller.py:88 IPKInstaller.py:96
msgid "Temp folder"
msgstr "Pasta temporária"

#: IPKInstaller.py:97
msgid "Select a package to install:"
msgstr "Selecionar o pacote a instalar:"

#: IPKInstaller.py:114
msgid "Are you ready to install ?"
msgstr "Está preparado para instalar?"

#: IPKInstaller.py:116
msgid "Install confirmation"
msgstr "Confirmação de Instalação"

#: IPKInstaller.py:124
msgid "Installing..."
msgstr "A instalar..."

#: IPKInstaller.py:127
msgid "Do you want to restart GUI now ?"
msgstr "Deseja reiniciar a GUI agora?"

#: IPKInstaller.py:129
msgid "Restart GUI."
msgstr "Reiniciar GUI."

#: IPKInstaller.py:157
msgid "Invert"
msgstr "Inverter"

#: IPKInstaller.py:158
msgid "Press OK to toggle the selection."
msgstr "Prima OK para mudar de seleção."

#: ImageManager.py:77 ImageManager.py:144 ImageManager.py:315 ImageManager.py:338
#: ImageManager.py:576 ui.py:45
msgid "Image manager"
msgstr "Gestão de Imagens"

#: ImageManager.py:100
msgid "Flash"
msgstr "Flash"

#: ImageManager.py:104 ImageManager.py:938
msgid "Downloads"
msgstr "Descargas"

#: ImageManager.py:222 ImageManager.py:229 ImageManager.py:249
msgid "Select an image to flash:"
msgstr "Selecione uma imagem para Flashar:"

#: ImageManager.py:229
msgid "Device: None available"
msgstr "Dispositivo: Não disponível"

#: ImageManager.py:249
msgid "Free space:"
msgstr "Espaço livre:"

#: ImageManager.py:249 MountManager.py:179 MountManager.py:375
msgid "MB"
msgstr "MB"

#: ImageManager.py:295
msgid ""
"Are you sure you want to delete this image backup:\n"
" "
msgstr ""
"Tem a certeza que prentende apagar este Backup:\n"
" "

#: ImageManager.py:299
msgid "You have no image to delete."
msgstr "Não tem imagens para apagar."

#: ImageManager.py:326
msgid ""
"Do you want to create a full image backup?\n"
"This can take about 6 minutes to complete."
msgstr ""
"Deseja criar um backup de imagem completo?\n"
"Isso pode levar cerca de 6 minutos até ficar concluído."

#: ImageManager.py:328
msgid "Backup confirmation"
msgstr "Confirmação de Backup"

#: ImageManager.py:357
msgid ""
"Are you sure you want to flash this image:\n"
" "
msgstr ""
"Tem certeza de que deseja Flashar esta imagem:\n"
" "

#: ImageManager.py:359
msgid "Restore confirmation"
msgstr "Confirmação de restauro"

#: ImageManager.py:361
msgid "You have no image to flash."
msgstr "Não tem imagem para Flashar."

#: ImageManager.py:371
msgid "Please wait while the flash prepares"
msgstr "Por favor, aguarde enquanto o flash é preparado"

#: ImageManager.py:393
#, python-format
msgid ""
"ofgwrite error (also sent to any debug log):\n"
"%s"
msgstr "erro de ofgwrite (também enviado para qualquer log de depuração): %s"

#: ImageManager.py:473
#, python-format
msgid ""
"Your %s %s is about to create a full image backup, this can take about 6 minutes "
"to complete.\n"
"Do you want to allow this?"
msgstr ""
"A sua %s %s está prestes a executar um backup de imagem completo, isto pode levar "
"cerca de 6 minutos para completar,\n"
"Quer permitir isto?"

#: ImageManager.py:577
msgid "Setting up..."
msgstr "A configurar ..."

#: ImageManager.py:580
msgid "Checking free RAM.."
msgstr "A verificar RAM livre..."

#: ImageManager.py:583
msgid "Creating swap.."
msgstr "A criar Swap..."

#: ImageManager.py:586 ImageManager.py:589
msgid "Backing up kernel..."
msgstr "A efetuar Backup do kernel..."

#: ImageManager.py:592 ImageManager.py:595
msgid "Backing up root file system..."
msgstr "A efetuar Backup do sistema de ficheiros..."

#: ImageManager.py:598 ImageManager.py:601
msgid "Removing temp mounts..."
msgstr "A remover montagens temporárias..."

#: ImageManager.py:604 ImageManager.py:607
msgid "Moving to backup Location..."
msgstr "A mover para o local de backup..."

#: ImageManager.py:610 ImageManager.py:613
msgid "Creating zip..."
msgstr "A criar zip..."

#: ImageManager.py:616
msgid "Backup complete..."
msgstr "Backup completo..."

#: ImageManager.py:635
msgid "The backup location does not have enough free space.\n"
msgstr "O local de backup não tem espaço livre suficiente.\n"

#: ImageManager.py:671
msgid ""
"Sorry, not enough free ram found, and no physical devices that supports SWAP "
"attached. Can't create Swapfile on network or fat32 filesystems, unable to make "
"backup"
msgstr ""
"Desculpe, não há memória RAM suficiente, e sem dispositivos físicos que suportam "
"ficheiros SWAP. Não é possível criar o ficheiro SWAP em rede ou fat32 e, não é "
"possível efetuar o backup"

#: ImageManager.py:951
msgid "Select an image to Download:"
msgstr "Selecionar a imagem para Descarga:"

#: ImageManager.py:953
msgid "Download"
msgstr "Descarregar"

#: ImageManager.py:1088
msgid ""
"Are you sure you want to download this image:\n"
" "
msgstr ""
"Deseja fazer a descarga desta imagem:\n"
" "

#: ImageManager.py:1090
msgid "Download confirmation"
msgstr "Confirmação de download"

#: ImageManager.py:1092
msgid "You have no image to download."
msgstr "Não tem imagens para descarregar."

#: ImageManager.py:1101
msgid "Downloading"
msgstr "A transferir..."

#: MountManager.py:27 ui.py:53
msgid "Mount manager"
msgstr "Gestor de Montagens"

#: MountManager.py:48
msgid "Setup mounts"
msgstr "Configurar Montagens"

#: MountManager.py:49
msgid "Un-mount"
msgstr "Desmontar"

#: MountManager.py:50
msgid "Mount"
msgstr "Montagem"

#: MountManager.py:80
msgid "Use as HDD"
msgstr "Usar como HDD"

#: MountManager.py:101
msgid "Please wait while scanning for devices..."
msgstr "Aguarde por favor enquanto se procura por dispositivos..."

#: MountManager.py:130 MountManager.py:343
msgid "HARD DISK: "
msgstr "DISCO RÍGIDO: "

#: MountManager.py:139 MountManager.py:352
msgid "USB: "
msgstr "USB: "

#: MountManager.py:155 MountManager.py:157 MountManager.py:167 MountManager.py:169
#: MountManager.py:358
msgid "None"
msgstr "Nenhum"

#: MountManager.py:156 MountManager.py:181 MountManager.py:359 MountManager.py:377
msgid "unavailable"
msgstr "indisponível"

#: MountManager.py:175 MountManager.py:177 MountManager.py:179 MountManager.py:181
#: MountManager.py:371 MountManager.py:373 MountManager.py:375 MountManager.py:377
msgid "Size: "
msgstr "Tamanho: "

#: MountManager.py:175 MountManager.py:371
msgid "TB"
msgstr "TB"

#: MountManager.py:177 MountManager.py:373
msgid "GB"
msgstr "GB"

#: MountManager.py:189 MountManager.py:203 MountManager.py:223
msgid "Mount: "
msgstr "Montagem: "

#: MountManager.py:189
msgid "Type: "
msgstr "Tipo: "

#: MountManager.py:214
msgid "Mount failed"
msgstr "Montagem fallhou"

#: MountManager.py:233
msgid ""
"Can't un-mount the partition, make sure it is not being used for swap or record/"
"timeshift paths"
msgstr ""
"Não pode desmontar a partição, certifique-se que não está a ser usada para swap "
"ou caminhos de registo/timeshift"

#: MountManager.py:251
msgid "This device is already mounted as HDD."
msgstr "Este dispositivo já está montado como HDD."

#: MountManager.py:286
msgid "Choose where to mount your devices to:"
msgstr "Escolha onde montar os seus dispositivos:"

#: MountManager.py:300
#, python-format
msgid "Please wait while scanning your %s %s devices..."
msgstr "Espere por favor durante a verificação de seus dispositivos de %s %s..."

#: MountManager.py:404
msgid "Updating mount locations."
msgstr "A Atualizar os locais de montagem."

#: MountManager.py:406
msgid "Please wait."
msgstr "Por favor, aguarde."

#: MountManager.py:409
#, python-format
msgid ""
"The changes need a system restart to take effect.\n"
"Restart your %s %s now?"
msgstr ""
"As modificações necessitam de um reinício para serem aplicadas.\n"
"Reiniciar a sua %s %s agora?"

#: MountManager.py:411
#, python-format
msgid "Restart %s %s."
msgstr "Reiniciar %s  %s."

#: RestoreWizard.py:101
msgid "OK, to perform a restore"
msgstr "OK, para executar o restauro"

#: RestoreWizard.py:101
msgid "Exit the restore wizard"
msgstr "Sair do assistente de restauro"

#: RestoreWizard.py:105
msgid "YES, to restore settings"
msgstr "Sim, para restaurar as definições"

#: RestoreWizard.py:105
msgid "NO, do not restore settings"
msgstr "NÃO, não restaurar definições"

#: RestoreWizard.py:111 RestoreWizard.py:114
msgid "YES, to restore plugins"
msgstr "Sim, para restaurar plugins"

#: RestoreWizard.py:112 RestoreWizard.py:115
msgid "NO, do not restore plugins"
msgstr "NÃO, não restaurar plugins"

#: RestoreWizard.py:139 RestoreWizard.py:144
msgid "Please wait while the system gathers information..."
msgstr "Por favor, aguarde enquanto o sistema reúne informações..."

#: RestoreWizard.py:140 RestoreWizard.py:145 RestoreWizard.py:152
#: RestoreWizard.py:161 RestoreWizard.py:170 RestoreWizard.py:174
#: RestoreWizard.py:199 RestoreWizard.py:203 RestoreWizard.py:212
#: RestoreWizard.py:275 RestoreWizard.py:279 RestoreWizard.py:283 plugin.py:137
msgid "Restore wizard"
msgstr "Assistente de Restauro"

#: RestoreWizard.py:151
msgid "Please wait while plugins restore completes..."
msgstr "Por favor aguarde até ao restauro completo dos plugins..."

#: RestoreWizard.py:160
msgid ""
"Sorry the feeds are down for maintenance, Please try using Backup Manager to "
"restore plugins later."
msgstr ""
"Desculpe, os Feeds estão em manutenção, por favor tente  usar o Gestor de Backups "
"para restaurar mais tarde os plugins."

#: RestoreWizard.py:169
#, python-format
msgid ""
"Your %s %s is not connected to the internet, Please try using Backup Manager to "
"restore plugins later."
msgstr ""
"A sua %s %s não está conetada à internet, por favor, tente usar o Gestor de "
"Backup para restaurar mais tarde os plugins."

#: RestoreWizard.py:211
msgid "Please wait while settings restore completes..."
msgstr "Por favor aguarde até ao restauro completo das definições..."

#: RestoreWizard.py:274
#, python-format
msgid ""
"Your %s %s is not connected to the internet, Please try using the Backup Manager "
"to restore plugins later."
msgstr ""
"A sua %s %s não está conetada à internet, por favor, tente usar o Gestor de "
"Backup para restaurar os plugins mais tarde."

#: RestoreWizard.py:278
msgid ""
"Sorry the feeds are down for maintenance, Please try using the Backup Manager to "
"restore plugins later."
msgstr ""
"Desculpe, os Feeds estão em manutenção, por favor tente  usar o Gestor de Backups "
"para restaurar mais tarde os plugins."

#: SwapManager.py:81 ui.py:61
msgid "Swap manager"
msgstr "Gestor de  Swap"

#: SwapManager.py:99
msgid "Swap place:"
msgstr "Local Swap:"

#: SwapManager.py:101
msgid "Swap size:"
msgstr "Tamanho Swap:"

#: SwapManager.py:103
msgid "Status:"
msgstr "Estado:"

#: SwapManager.py:104 SwapManager.py:237
msgid "Inactive"
msgstr "Inativo"

#: SwapManager.py:105 SwapManager.py:232
msgid "Active"
msgstr "Ativo"

#: SwapManager.py:107 SwapManager.py:236
msgid "Activate"
msgstr "Ativar"

#: SwapManager.py:108 SwapManager.py:178
msgid "Create"
msgstr "Criar"

#: SwapManager.py:109
msgid "Autostart"
msgstr "Início automático"

#: SwapManager.py:137 SwapManager.py:232 SwapManager.py:237
msgid "Current status:"
msgstr "Estado atual:"

#: SwapManager.py:138
msgid "Wait please while scanning..."
msgstr "Por favor, aguarde enquanto procura..."

#: SwapManager.py:231
msgid "Deactivate"
msgstr "Desativar"

#: SwapManager.py:238
msgid "Enable swap at startup"
msgstr "Ativar Swap no arranque"

#: SwapManager.py:252
msgid ""
"Swap file not found. You have to create the file before you try to activate it."
msgstr "Ficheiro de Swap não encontrado. Tem de criar o ficheiro, antes de ativar."

#: SwapManager.py:253 SwapManager.py:327
msgid "Info"
msgstr "Informação"

#: SwapManager.py:287
msgid "Please select device to use as swapfile location"
msgstr "Por favor, selecione o dispositivo para usar como swapfile"

#: SwapManager.py:289
msgid ""
"Sorry, no physical devices that supports swap attached. Can't create Swap file on "
"network or fat32 filesystems"
msgstr ""
"Desculpe, não há dispositivos que suportam Swap. Não é possível criar o ficheiro "
"SWAP em sistemas de arquivos de rede ou fat32"

#: SwapManager.py:294
msgid "8 Mb"
msgstr "8 Mb"

#: SwapManager.py:295
msgid "16 Mb"
msgstr "16 Mb"

#: SwapManager.py:296
msgid "32 Mb"
msgstr "32 Mb"

#: SwapManager.py:297
msgid "64 Mb"
msgstr "64 Mb"

#: SwapManager.py:298
msgid "96 Mb"
msgstr "96 Mb"

#: SwapManager.py:299
msgid "128 Mb"
msgstr "128 MB"

#: SwapManager.py:300
msgid "256 Mb"
msgstr "256 MB"

#: SwapManager.py:301
msgid "Select the Swap File Size:"
msgstr "Selecionar o tamanho de Swap:"

#: SwapManager.py:306
msgid "Wait please while creating swapfile..."
msgstr "Por favor, aguarde enquanto o ficheiro Swap é criado..."

#: SwapManager.py:326
msgid "You have to create a swap file before trying to activate the autostart."
msgstr "Tem de criar um ficheiro Swap, antes de ativar o Início Automático."

#: plugin.py:126
msgid "Install extensions."
msgstr "Instalar extensões."

#: plugin.py:132 ui.py:57
msgid "Script runner"
msgstr "Executor de scripts"

#: plugin.py:138
msgid "Ipkg"
msgstr "Ipkg"

#: plugin.py:139
msgid "ViX Backup manager"
msgstr "Gestor de Backup OBH"

#: plugin.py:140
msgid "ViX Image manager"
msgstr "Gestor da Imagem OBH"

#: plugin.py:141
msgid "ViX Mount manager"
msgstr "Gestor de montagem OBH"

#: plugin.py:142
msgid "ViX Script runner"
msgstr "Executor de scripts OBH"

#: plugin.py:143
msgid "ViX Swap manager"
msgstr "Gestor de Swap OBH"

#: ui.py:17
msgid "OBH"
msgstr "OBH"

#: ui.py:18
msgid "Main menu"
msgstr "Menu Principal"

#: ui.py:18
msgid "Setup"
msgstr "Definições"

#: ui.py:42
msgid "Manage the backups of your settings."
msgstr "Gerir os backups das suas definições."

#: ui.py:46
msgid "Create and flash complete images of your system."
msgstr "Criar e restaurar imagens completas do sistema."

#: ui.py:49
msgid "Install local extension"
msgstr "Instalar extensão local"

#: ui.py:50
msgid "Install IPK's from your tmp folder."
msgstr "Instalar IPK´s a partir da pasta /tmp."

#: ui.py:54
msgid "Manage your devices mount points."
msgstr "Escolha onde montar os seus dispositivos."

#: ui.py:58
msgid "Run your shell scripts."
msgstr "Executar scripts Shell."

#: ui.py:62
msgid "Create and Manage your swap files."
msgstr "Criar e Gerir os ficheiros de Swap."

msgid "Image manager settings"
msgstr "Configuração do Gestor de Imagens"

msgid "Automatic settings backup"
msgstr "Backup automático das definições"

msgid "Backup location"
msgstr "Local para Backups"

msgid "Backup prefix"
msgstr "Prefixo do Backup"

msgid "Schedule backups"
msgstr "Agendar backups"

msgid "Create a settings backup before updating."
msgstr "Crie um backup de configurações antes de atualizar."

msgid "Location of where the image backup should be saved."
msgstr "Local de onde o backup de imagem deve ser salvo."

msgid "Allows you to tag your image backup to a box. (default is box name/type)"
msgstr ""
"Permite-lhe etiquetar o backup da imagem com o nome da STB. (padrão é o nome/tipo)"

msgid "Allows you to schedule your image backups."
msgstr "Permite agendar os seus backups de imagem."
